src/main/java/uk/gov/hmcts/reform/userprofileapi/infrastructure/clients/IdamUserResponse.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.hmcts.reform.userprofileapi.infrastructure.clients;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import javax.validation.constraints.NotNull;
+

```

Comment:
Use lombok @Getter instead of having all these getters

Reply:
Done